### DISCUSSION TITLE
Xamarin.UITest alternative for MAUI

### DISCUSSION THREAD

**Discussion by listepo** [dotnet/maui#1134] [Q&A]
Are there alternatives like Xamarin.UITest for MAUI?

**Comment by dansiegel** [comment]
you might check out [Xappium](https://github.com/Xappium/xappium.uitest) which we've been working on to ultimately support Forms and Maui

**Reply by listepo** [reply]
Nice, thanks

**Reply by listepo** [reply]
@dansiegel looks like Xappium is no longer being developed. I'm also interested in E2E for .NET MAUI Blazor app, @Redth maybe you have information.
